package.json

```diff
@@ -27,8 +32,35 @@
   "homepage": "https://github.com/dmorosinotto/ng-metadata-quickstart#readme",
   "dependencies": {
     "angular": "^1.5.3",
-    "ng-metadata": "^1.5.0",
+    "ng-metadata": "^1.5.2",
     "reflect-metadata": "^0.1.3",
-    "systemjs": "^0.19.26"
+    "system-text": "^0.1.0",
+    "systemjs": "^0.19.26",
+    "systemjs-plugin-css": "^0.1.20",
+    "systemjs-plugin-text": "git+https://github.com/systemjs/plugin-text.git",
+    "traceur": "0.0.107"
```

Comment:
@Oipo can you explain me why you need the traceur dependency?
I see that you reference it in the systemjs.conf.js map, but what is it used for?  
The ES6 compilation is done by TS isn't it? Or am I loosing something? Or is it needed for running the test framework?